src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import React, { type ComponentType } from 'react';
+import { spacersNum, colors } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core';
+import type { Props } from '../WidgetFeedback.types';
+
+const styles = {
```

Comment:
Hey,
Can you help me understand why you used `5px`, `14px`, `15px` instead of the [SpacerNum const ](https://ui.dhis2.nu/demo/?path=/story/using-ui-useful-constants--page)?

Thank you!

Reply:
Not to familiar with dhis/ui, but will be sure to study it in the coming days! 
I just used the pixel-sizes from the sketch-inspector. I've replaced most of them with the nearest SpacerNum-constant.
(Did not seem to work on font-sizes, but I will investigate it further)
Thanks!